.github/workflows/ci.yaml

```diff
@@ -81,6 +81,7 @@ jobs:
           - tls-integration
           - backup-integration
           - metric-integration
+          - sharding-integration
```

Comment:
Do we really want to add sharding test to CI now?

Reply:
yes